### ISSUE TITLE
OSG?

### ISSUE THREAD

**Issue by SaiRaul** [bjornblissing/osgueye#1]
What you mean that osg and includeing files?

**Comment by SaiRaul** [comment]
How to use that in my project?

**Comment by bjornblissing** [comment]
That is depending on your platform. If you run Linux you can install OpenSceneGraph:
`sudo apt-get install openscenegraph`

For windows there are pre-built binaries here:
https://objexx.com/OpenSceneGraph.html

There is always the option to download the source from the link in my previous message and built it you self.

**Comment by SaiRaul** [comment]
Yeah im a window user and  i have done download and add it's including files and lib files also but still i didn't find osg/program and osg/shader files.

**Comment by bjornblissing** [comment]
I need some more information regarding the kinds of errors you see:

1. Are you running CMake?
2. Do you get errors during the CMake Configuration stage?
3. Errors during the CMake Generation stage?
3. What compiler are you using? Do you get compiler errors?
4. If the compilation is working, do you get runtime errors?

**Comment by SaiRaul** [comment]
Hai bro now i'm getting exe now and no error also but i have get live  in my ui design can u please help me ?
